spec/rails_helper.rb

```diff
@@ -96,5 +96,6 @@
 
   FactoryBot::SyntaxRunner.class_eval do
     include RSpec::Matchers
+    include RSpec::Mocks::ExampleMethods
```

Comment:
FactoryBot couldn't use the `allow()` method from rspec-mocks unless we add this include.